src/components/Header/Header.js

```diff
@@ -30,13 +30,16 @@ const Header = React.memo(function Header({ compactMode, toggleMenuPanel }) {
       <div
         css={`
           display: flex;
-          flex-grow: 1;
-          justify-content: center;
+          align-self: stretch;
+
+          & > * {
+            align-self: stretch;
+          }
         `}
       >
-        {!compactMode && <Clock />}
+        {!compactMode && <ClockModule />}
+        <Account />
```

Comment:
Should we rename this `AccountModule` for consistency?

Reply:
Yep good idea